scripts/travis.sh

```diff
@@ -309,6 +323,25 @@ handle_sdk_after_failure_phase()
   do_default_after_failure_actions
 }
 
+#######################################################################
+# Format build variant ################################################
+#######################################################################
+
+handle_format_script_phase()
+{
+  make clang-format && check_format_errors
+}
+
+handle_format_after_success_phase()
+{
+ pass
```

Comment:
Is this a real function/builtin?  Can't find it on my system.  Also clang-formatter would be mad at you, missing some whitespace here.

Reply:
yeah that was a whoops, will fix